airbyte-webapp/.stylelintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": [
+    "stylelint-config-standard",
+    "stylelint-config-standard-scss",
+    "stylelint-config-css-modules",
+    "stylelint-config-prettier-scss"
+  ],
+  "rules": {
+    "selector-class-pattern": null,
+    "color-function-notation": null,
+    "font-family-name-quotes": null,
+    "scss/dollar-variable-empty-line-before": null,
+    "scss/dollar-variable-pattern": ".*",
+    "scss/percent-placeholder-pattern": ".*",
```

Comment:
This enforces kebab case in s/css class names, which doesn't work well for us thanks to modules.